docs/sources/articles/networking.md

```diff
@@ -363,17 +363,19 @@ to provide special options when invoking `docker run`.  These options
 are covered in more detail in the [Docker User Guide](/userguide/dockerlinks)
 page.  There are two approaches.
 
-First, you can supply `-P` or `--publish-all=true|false` to `docker run`
-which is a blanket operation that identifies every port with an `EXPOSE`
-line in the image's `Dockerfile` and maps it to a host port somewhere in
-the range 49153–65535.  This tends to be a bit inconvenient, since you
-then have to run other `docker` sub-commands to learn which external
-port a given service was mapped to.
-
-More convenient is the `-p SPEC` or `--publish=SPEC` option which lets
-you be explicit about exactly which external port on the Docker server —
-which can be any port at all, not just those in the 49153-65535 block —
-you want mapped to which port in the container.
+First, you can supply `-P` or `--publish-all=true|false` to `docker run` which
+is a blanket operation that identifies every port with an `EXPOSE` line in the
+image's `Dockerfile` and maps it to a host port somewhere within an *ephemeral
+port range*. This tends to be a bit inconvenient, since you then have to run
```

Comment:
I'd ditch the `inconvenient` phrase, and rewrite that sentence to tell the user to use the `docker port` command to find out what port.

This functionality has important uses that are not about convenience (like running multiple containers that would otherwise conflict)